dll/win32/mstask/stubs.c

```diff
@@ -0,0 +1,133 @@
+
+#include "precomp.h"
+
+WINE_DEFAULT_DEBUG_CHANNEL(mstask);
+
+#define NET_API_STATUS DWORD
+typedef HANDLE ATSVC_HANDLE, SASEC_HANDLE;
+typedef struct _AT_INFO *LPAT_INFO;
+typedef struct _AT_ENUM_CONTAINER *LPAT_ENUM_CONTAINER;
```

Comment:
Where is `_AT_ENUM_CONTAINER` defined?

Reply:
at_svc_c.h, which is autogenerated in netapi folder.